src/classes/BGE_BatchGiftEntry_UTIL.cls

```diff
@@ -36,6 +36,82 @@
 */
 public with sharing class BGE_BatchGiftEntry_UTIL {
 
+    /*******************************************************************************************************
+    * @description returns a list of DataImport__c fields the Batch Gift Entry UI needs in SOQL
+    * @param batchId the batch for which to get all fields for soql
+    * @return List<String> list of DataImport__c field api names
+    */
+    public static List<String> getDataImportFields(Id batchId) {
```

Comment:
What's the thought behind having these overloaded methods? Does this follow a certain pattern? I found it difficult to read, but if this is just a larger convention then it might just be me!

Reply:
renamed one for clarity - it didn't need to have the same name. the other two just provide some defaulting.